docs/section5/Rule5-19.md

```diff
@@ -36,12 +36,12 @@
 
 **Rule Assertion:**
 
-- Case 1: If all building segments with building areas not shown in Table G3.1.1-1 are new, and the total window-wall-ratio is equal to that in P_RMR or 40%, whichever is smaller: `if ( NOT manual_check_flag ) AND ( window_wall_areas_dictionary_b["NONE"] == min(window_wall_areas_dictionary_p["NONE"], 0.4) ): PASS`
+- Case 1: If all building segments with building areas not shown in Table G3.1.1-1 are new, and the total window-wall-ratio is equal to that in P_RMR or 40%, whichever is smaller: `if ( NOT manual_check_flag ) AND ( window_wall_areas_dictionary_b["OTHER"] == min(window_wall_areas_dictionary_p["OTHER"], 0.4) ): PASS`
```

Comment:
@weilixu  @xingcx  I believe the logic for Case 1 should be:
```
total_window_area_b = window_wall_areas_dictionary_b["OTHER"]["total_window_area"]
total_window_area_p = window_wall_areas_dictionary_p["OTHER"]["total_window_area"]
total_wall_area_b = window_wall_areas_dictionary_b["OTHER"]["total_wall_area"]

if (Not manual_check_flag) AND (total_window_area_b == min(total_window_area_p,  total_wall_area_b * 0.4): PASS
```
And similar changes for the other cases.

Reply:
@weilixu @xingcx  Please let me know how you think. If you guys approve this, I will update the RDS based on Charlie's comment.